assignmentTest/splab/Gopistol/invitation/src/main/java/com/splab/invitation/domain/member/service/MemberService.java

```diff
@@ -0,0 +1,60 @@
+package com.splab.invitation.domain.member.service;
+
+import com.splab.invitation.domain.member.entity.Member;
+import com.splab.invitation.domain.member.service.dto.InviteMemberCommand;
+import com.splab.invitation.persistence.member.MemberRepository;
+import java.time.Instant;
+import java.util.concurrent.ConcurrentHashMap;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class MemberService {
+
+  private final EmailService emailService;
+  private final MemberRepository repository;
+  private static final Integer KEY_MAX_LENGTH = 6;
+  private static final Integer SLICE_INDEX = 0;
+  private static final ConcurrentHashMap<String, String> db = new ConcurrentHashMap<>();
+
+  public String create(InviteMemberCommand command) {
+    Member member = command.toEntity();
+    repository.save(member);
+
+    String passKey = generatePassKey();
+    String email = command.getEmail();
+
+    db.put(email, passKey);
+
+    emailService.sendEmail(command, passKey);
+
+    return email;
+  }
+
+  public void join(String email, String key) {
+    String passKey = db.get(email);
+
+    if (passKey.equals(key)) {
```

Comment:
사용자의 편의를 위해서 ignore case를 고려할 수도 있을 것 같아요!
```suggestion
    if (passKey.equalsIgnoreCase(key)) {
```